.fly.yml

```diff
@@ -2,13 +2,13 @@ app: cdn
 config:
   flyCDN:
     backends:
-      getting-started:
+      fly:
         type: origin
-        origin: https://getting-started.edgeapp.net
+        origin: https://fly.io/
         headers:
-          host: getting-started.edgeapp.net
+          host: fly.io
     rules:
     - actionType: rewrite
-      backendKey: getting-started
+      backendKey: fly
     middleware:
-    - type: https-upgrader
+    - type: auto-webp
```

Comment:
For this to work, you need to change `src/config/` to map `auto-webp` to the right middleware.